packages/cli/src/commands/pg/credentials/url.ts

```diff
@@ -0,0 +1,70 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import {legacyEssentialPlan} from '../../../lib/pg/util'
+import {getAddon} from '../../../lib/pg/fetcher'
+import pgHost from '../../../lib/pg/host'
+import {URL} from 'url'
+import type {CredentialsInfo} from '../../../lib/pg/types'
+import heredoc from 'tsheredoc'
+
+export default class Url extends Command {
+  static topic = 'pg';
+  static description = 'show information on a database credential';
+  static flags = {
+    name: flags.string({
+      char: 'n',
+      description: 'which credential to show (default credentials if not specified)',
+      default: 'default',
+    }),
+    app: flags.app({required: true}),
+  };
```

Comment:
Missing `remote` flag:

```suggestion
    remote: flags.remote(),
  }
```